main.go

```diff
@@ -139,22 +148,11 @@ func Remove(phonebookFile string) {
 			if nosure == 1 {
```

Comment:
I think `nosure` is always `0` in this context. You probably forgot to read it.

Reply:
![photo_2022-01-26_11-53-21](https://user-images.githubusercontent.com/81423410/151142045-e8ad7155-1cb4-49b2-ae7c-885107a5276d.jpg)
![photo_2022-01-26_11-53-33](https://user-images.githubusercontent.com/81423410/151142052-83e23c55-6f5a-4189-bc0e-bf41a6d0953a.jpg)

Reply:
Its working when i run my code